src/objects.h

```diff
@@ -28,7 +28,7 @@
 
 /****************************************************************************
 **
-*T  Obj . . . . . . . . . . . . . . . . . . . . . . . . . . . type of objects
+*t  Obj . . . . . . . . . . . . . . . . . . . . . . . . . . . type of objects
```

Comment:
Are the valid letters and their meaning documented somewhere?

Reply:
No.
As far as I know, the whole markup in `src` files is not defined anywhere.
It would make sense to document this
if the current format or some extension of it would be kept for the future.
This is one of the questions which I had raised in a message to gap@gap-system.org,
concerning an official documentation for the GAP C API.